src/dodal/beamlines/i10.py

```diff
@@ -260,45 +284,63 @@ def idd_la_angle(
 
 
 @device_factory()
-def m1() -> Mirror_with_piezo:
-    return Mirror_with_piezo(prefix="BL10I-OP-COL-01:")
+def mirror_1() -> PiezoMirror:
+    return PiezoMirror(prefix=f"{PREFIX.beamline_prefix}-OP-COL-01:")
 
 
 @device_factory()
-def m3m5() -> Mirror_with_piezo:
-    return Mirror_with_piezo(prefix="BL10I-OP-SWTCH-01:")
+def mirror_3_5() -> PiezoMirror:
```

Comment:
What does 3_5 mean here and is it obvious to the beamline user? (I am not familiar with the optical components for the beamline)

Reply:
Looking at the PV prefixes, are they a Collimating mirror, mirror for switching... something, and a focussing mirror? If they have different purposes may be useful to name them as such